lib/string.gd

```diff
@@ -860,6 +860,73 @@ BindGlobal("BHINT", MakeImmutable("\>\<"));
 
 DeclareGlobalFunction("StringOfMemoryAmount");
 
+#############################################################################
+##
+##  <#GAPDoc Label="StringFormatted">
+##  <ManSection>
+##  <Func Name="StringFormatted" Arg='string, data...'/>
+##  <Func Name="PrintFormatted" Arg='string, data...'/>
+##  <Func Name="PrintToFormatted" Arg='stream, string, data...'/>
+##
+##  <Description>
+##  These functions format an input string.
+##  <Ref Func="StringFormatted"/> returns the formatted string,
+##  <Ref Func="PrintFormatted"/> prints the formatted string and
+##  <Ref Func="PrintToFormatted"/> appends the formatted string to <A>stream</A>,
+##  which can be either an output stream or a filename.
+##  <P/>
+##  <A>data</A> is a list of values to printed, using the formatting rules below:
```

Comment:
Also, I think this can be confusing: the user might think that the second (resp third, in case of `PrintToFormatted`) argument should be a list, containing the values. Perhaps like this:
> All arguments after `<A>string</A>` form the list `<A>data</A>` of values to be printed, ...